cmd/avalanche-network-runner/server/server.go

```diff
@@ -22,25 +22,28 @@ func init() {
 }
 
 var (
-	logLevel    string
-	port        string
-	gwPort      string
-	gwDisabled  bool
-	dialTimeout time.Duration
+	logLevel           string
+	port               string
+	gwPort             string
+	gwDisabled         bool
+	dialTimeout        time.Duration
+	disableNodesOutput bool
```

Comment:
Why do we want to disable node log output?

Reply:
This is a fast server option. Makes fast filtering of logs using process control output and not avalanchego/pluging configs. Think it is useful for code iteration, to just avoid msgs and be able to fast look at the one you want.